wouso/games/teamquest/models.py

```diff
@@ -14,7 +15,11 @@ def is_head(self):
 
 
 class TeamQuest(models.Model):
-	pass
+    start = models.DateTimeField()
```

Comment:
Add `null`, `blank` and, if required, `default` parameter. Rename to `start_time`.